test/gpb_parse_tests.erl

```diff
@@ -770,13 +770,16 @@ can_prefix_record_names_by_proto_test() ->
     {ok, Defs} = parse_lines(["enum    e1 {a=1; b=2;}",
                               "message m1 {required e1 f1=1;}",
                               "message m2 {required m1 f2=1;}",
+                              "message m3 {map<string, m1> m=1;}",
                               "service s1 {",
                               "  rpc req(m1) returns (m2) {};",
                               "}",
                               "extend m1 { optional uint32 fm2=2; }"]),
     Defs1 = [{{msg_containment, "proto1"}, [['.',m1]]},
              {{msg_containment, "proto2"}, [['.',m2]]} | Defs],
     [{{enum,e1},  [{a,1},{b,2}]}, %% not prefixed
+     {{msg,m3},
+      [{field,m,1,2,{map,string,{msg,p1_m1}},repeated,[]}]},
```

Comment:
Hmm, sorry, ... I mean it should be `[#?gpb_field{...}]` instead of `[{field,...}]`